dkrichards86/gotodo
# Tablewriter

### Summary

This branch formats CLI output using [tablewriter](https://github.com/olekukonko/tablewriter)

### Related Issues

N/A

### Verification Steps

1. `go run ./cmd/gotodo/... ls`
2. `go run ./cmd/gotodo/... add <msg>`
3. `go run ./cmd/gotodo/... edit <id> <msg>`
4. Observe tabular format
